### PR TITLE
Created Homepage

### LINKED ISSUE #9: Stworzyć Homepage
Strona główna powinna być na roucie `/`. Powinno to być w stylu strony głownej produktu SaaS.

### PR DESCRIPTION
**Pull request by hubcio2115**
closes #9

### CODE REVIEW

**Review by hubcio2115 — COMMENTED** [review]
LGTM

File: src/components/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Monitor, Moon, Sun } from "lucide-react";
+
+import { Toggle } from "./ui/toggle";
+
+export default async function Footer() {
```
[the remark this answers is not in the document]
Comment (Ciszkoo) [reply src/components/footer.tsx]: @hubcio2115 is it async for any reason?

[the remark this answers is not in the document]
Comment (hubcio2115) [reply src/components/footer.tsx]: No, actually

**Review by Ciszkoo — APPROVED** [review]
lgtm ;)

**Review by Maxxxxxxxxxxxxxxxxxxxxx — APPROVED** [review]

**Pull request merged**